authors/apps/articles/tests/test_create_article.py

```diff
@@ -198,3 +200,35 @@ def test_to_fetch_non_exisiting_article(self):
                                    format='json')
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
         self.assertEqual('Not found.', response.data['detail'])
+
+    def test_favorite_article(self):
+        """Test to favorite an article"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        response = self.client.put(self.article_delete,
+                                   format='json',
+                                   HTTP_AUTHORIZATION='token {}'.format(token))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data['favorite'], True)
+
+    def test_unfavorite_a_favorited_article(self):
+        """Test to unfavorite an article that had previously been favorited"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        self.client.put(reverse('articles:articles-favorite',
```

Comment:
I think you should use the variable created to hold the url since you declared it above